### PR TITLE
feat: add new timescale and knative workshop

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: modules/timeseries-as-a-service/pages/timescale.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+will create two users: one for writing data, one for reading data. Just as you would do with a normal database:
+
+[source,sql]
+----
+CREATE USER pusher;
+ALTER USER pusher WITH PASSWORD 'pusher-password';
+GRANT INSERT ON ALL TABLES IN SCHEMA public TO pusher;
+
+CREATE USER dashboard;
+ALTER USER dashboard WITH PASSWORD 'dashboard-password';
```
Comment (clarkbw) [inline modules/timeseries-as-a-service/pages/timescale.adoc:110]:
```suggestion
CREATE USER dashboard WITH PASSWORD 'dashboard-password';
```
You can combine this into a single command.

File: modules/timeseries-as-a-service/pages/timescale.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+Pretty simple. There is a lot more to it, you can learn more about hypertables from the
+https://docs.timescale.com/api/latest/hypertable/[TimescaleDB documentation].
+
+Of course, we don't want to use the "admin" user with the different services which need access to the database. So we
+will create two users: one for writing data, one for reading data. Just as you would do with a normal database:
+
+[source,sql]
+----
+CREATE USER pusher;
+ALTER USER pusher WITH PASSWORD 'pusher-password';
```
Comment (clarkbw) [inline modules/timeseries-as-a-service/pages/timescale.adoc:106]:
```suggestion
CREATE USER pusher WITH PASSWORD 'pusher-password';
```
Same thing here.

**Review by clarkbw — COMMENTED** [review]
Looks great!

**Review by dejanb — APPROVED** [review]
Looks good!

**Pull request merged**